From 3ac6f8717a1e885e12fdb87aee67e6657db25002 Mon Sep 17 00:00:00 2001
From: hscham <hscham@chromium.org>
Date: Tue, 5 Apr 2022 10:34:34 +0900
Subject: [PATCH] Revert "Include a smaller UNIX header in
 build/build_config.h."

This reverts commit 1d647fd94e57291b9e09772422bb7336dad5aba6.
---
 build/build_config.h | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/build/build_config.h b/build/build_config.h
index 73730f92063e..9c6d17983921 100644
--- a/build/build_config.h
+++ b/build/build_config.h
@@ -99,7 +99,7 @@
 #define OS_LINUX 1
 #endif  // !defined(OS_CHROMEOS)
 // Include a system header to pull in features.h for glibc/uclibc macros.
-#include <assert.h>
+#include <unistd.h>
 #if defined(__GLIBC__) && !defined(__UCLIBC__)
 // We really are using glibc, not uClibc pretending to be glibc.
 #define LIBC_GLIBC 1
-- 
2.35.1.1094.g7c7d902a7c-goog

